### DISCUSSION TITLE
[done] Поле id

### DISCUSSION THREAD

**Discussion by alexrus79** [totumonline/totum-ru-issues-and-discussions#115] [Есть идея, а что если...]
Здравствуйте. 
Сделайте, пожалуйста, возможность скрывать значения поля id в таблицах. Чтобы делая свою нумерацию строк, нумерация поля id не вводила в заблуждение пользователя.

**Comment by ttm-alexeypolunin** [comment]
Принято

**Comment by ttm-alexeypolunin** [comment]
Готово в 2.3.39 — через `fieldhide` функции `setTableFormat`.

```
f1=: setTableFormat(fieldhide: 'id' = true)
```

Для роли `Создатель` всегда показываются. Для пользователя скрываются если указано. Возможность фильтровать по `id` тоже отключается:

<img width="518" src="https://user-images.githubusercontent.com/55755565/165509613-a64b2c1c-bc47-45fc-aaf6-66a3d722a4f0.png">

**Comment by alexrus79** [comment]
Спасибо!
